fintech-examples/fintech-impl/src/main/java/de/adorsys/opba/fintech/impl/service/BankSearchService.java

```diff
@@ -44,8 +47,15 @@ public InlineResponse2001 searchBank(String keyword, Integer start, Integer max)
 
     @SneakyThrows
     public InlineResponse2002 searchBankProfile(String bankId) {
+        UUID xRequestId = UUID.fromString(restRequestContext.getRequestId());
+
         return new InlineResponse2002().bankProfile(
-                ManualMapper.fromTppToFintech(tppBankSearchClient.bankProfileGET(tppProperties.getFintechID(),
-                        UUID.fromString(restRequestContext.getRequestId()), bankId).getBody().getBankProfileDescriptor()));
+                ManualMapper.fromTppToFintech(tppBankSearchClient.bankProfileGET(
+                        xRequestId,
+                        bankId,
+                        null,
```

Comment:
Please add a comment about why the values are null, or add some constant like:
`public static final LocalDateTime COMPUTE_X_TIMESTAMP_UTC = null`
and use it here instead of null

Reply:
3 constants were added, with comments regarding the meaning of null values